pkg/machine/hyperv/machine.go

```diff
@@ -683,3 +693,29 @@ func (m *HyperVMachine) setRootful(rootful bool) error {
 	m.HostUser.Modified = true
 	return nil
 }
+
+func (m *HyperVMachine) cleanupGVProxy() {
```

Comment:
It's not obvious to me when to call `Cleanup` vs `cleanup`.  Could the intended use be reflected in comments?

Reply:
fixed